x-pack/plugins/security_solution/server/search_strategy/timeline/factory/events/all/helpers.test.ts

```diff
@@ -42,12 +42,12 @@ describe('#formatTimelineData', () => {
             value: ['beats-ci-immutable-ubuntu-1804-1605624279743236239'],
           },
           {
-            field: 'source.geo.location',
-            value: [`{"lon":118.7778,"lat":32.0617}`],
+            field: 'threat.indicator.matched.field',
+            value: ['matched_field', 'other_matched_field', 'matched_field_2'],
           },
           {
-            field: 'threat.indicator.matched.field',
-            value: ['matched_field', 'matched_field_2'],
+            field: 'source.geo.location',
```

Comment:
The order here changed due to the addition of threat match fields to the ECS fields list, meaning it comes sooner than a requested field like this one (`source.geo.location`)